StyleChecker/src/bg/sofia/uni/fmi/mjt/stylechecker/analyzerTypes/AnalyzerType.java

```diff
@@ -0,0 +1,8 @@
+package bg.sofia.uni.fmi.mjt.stylechecker.analyzerTypes;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public interface AnalyzerType {
+    void analyze(String line, BufferedWriter output) throws IOException;
```

Comment:
Why do you need the BufferedWriter output? What if we want to show the output error somewhere else? In a browser for example.
 It would be better for the analyze() to return the error with some kind of error type and the error message.